### PR TITLE
rewrite deprecated attributes to work with Arduino 1.0.5 tools

### PR DESCRIPTION
**Pull request by joshgoebel**

### CODE REVIEW

File: src/Arduboy.h [hunk 1]
```diff
@@ -63,7 +63,7 @@ class Arduboy : public ArduboyCore
    */
   // void boot(); // defined in core.cpp
 
-  void start() __attribute__ ((deprecated("use begin() instead")));
+  void start() __attribute__((deprecated)) __attribute__ ((warning("use begin() instead")));
```
Comment (MLXXXp) [inline src/Arduboy.h:66]: I don't think you need to use `__attribute__` twice.
`void start() __attribute__ ((deprecated, warning("use begin() instead")));`

**Comment by joshgoebel** [comment]
Really? If that works submit a PR. :)  Didn' know that.

**Comment by MLXXXp** [comment]
> If that works submit a PR.

Yes, it works. I tried it.

**Comment by MLXXXp** [comment]
Created PR #129.

**Pull request merged**